### DISCUSSION TITLE
How does storage work for distributed enterprise

### DISCUSSION THREAD

**Discussion by cerberus10748** [Security-Onion-Solutions/securityonion#3246] [Unsupported Versions] [answered]
So I am working on gauging out my storage for my enterprise and have the following setup : Manager, Sensor 1 , Search Node, Sensor 2. Where are all the logs stored and where should I put the most storage? IE Does sensor 1 send all of its logs and data to the search node or manager node and only requires a smaller driver? Or does each sensor need a larger amount of storage to retain all alerts and logs?

**Comment by lock-wire** [comment, marked as answer]
You may find the architecture portion of the documentation helpful.  Here is the link to the system diagram and description for a distributed deployment.
https://docs.securityonion.net/en/latest/architecture.html#distributed

The hardware section also has a great discussion on hardware considerations by node.  He is the link to the area that fits your use case.
https://docs.securityonion.net/en/latest/hardware.html#manager-node-with-separate-search-nodes

The Logs portion of the documentation has a page on Ingest pipelines that will help as well.
https://docs.securityonion.net/en/latest/ingest.html#ingest

**Reply by cerberus10748** [reply]
So then according to this , the forward sensor would still store Pcaps and need large storage in addition to the search node storage correct?

**Reply by lock-wire** [reply]
Correct
